app/src/main/kotlin/edu/stanford/bdh/engagehf/messages/Message.kt

```diff
@@ -15,7 +15,6 @@ data class Message(
     val description: String? = null,
     val action: String?,
     val isDismissible: Boolean = true,
-    val isLoading: Boolean = false,
```

Comment:
There is some logic in `HomeViewModel` that shows a loading indicator while a message is being processed, e.g. health summary pdf generation. I think we still need this

Reply:
I found this out now, yes - but I believe this should actually be in some wrapper type or something, since it has nothing to do with the model itself. Like this, I would assume that the server sends us this information, which is not the case.

Reply:
Totally agree, in general there should be distinct types for domain and ui layer, e.g. `Message` and `MessageUiModel` and currently the message is serving for both layers